app/controllers/repos_controller.rb

```diff
@@ -27,10 +27,16 @@ def new
   end
 
   def show
-    @repo          = Repo.where(full_name: params[:full_name]).first
+    @repo = Repo.where(full_name: params[:full_name]).first
 
     if @repo
-      @docs        = @repo.doc_methods.order("created_at DESC").page(params[:page]).per_page(params[:per_page]||20)
+      @docs = @repo
+                .doc_methods
```

Comment:
Oh I like this style. Very smooth...
I haven't hit `#send`'s guts yet but I love the idea of the params being passed to a method named as such. Nice